lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -147,6 +147,41 @@ def create_or_update_build_phase(native_target, phase_name, phase_class = Xcodep
                 end
               end
           end
+
+          # Updates all target script phases for the current target, including creating or updating, deleting
+          # and re-ordering.
+          #
+          # @return [void]
+          #
+          def create_or_update_user_script_phases(script_phases, native_target)
```

Comment:
this was moved to become a class method so that `pod_target_integrator.rb` can use it also, same logic as it has been done before.